feature_tests/steps/common.py

```diff
@@ -0,0 +1,91 @@
+from uuid import UUID
+
+import parse
+from behave import register_type
+
+
+@parse.with_pattern(
+    r"\{[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12}\}"
```

Comment:
i'd factor out that regex into a named constant - because it's not totally clear what it is (i mean i know its a uuid, but still)